code/modules/mob/living/carbon/brain/posibrain.dm

```diff
@@ -78,17 +78,13 @@ var/global/posibrain_notif_cooldown = 0
 	return
 
 /obj/item/device/mmi/posibrain/proc/transfer_personality(var/mob/candidate)
-	if(brainmob && brainmob.key) //Prevents hostile takeover if two ghosts get the prompt for the same brain.
+	if(brainmob.key) //Prevents hostile takeover if two ghosts get the prompt for the same brain.
 		candidate << "This brain has already been taken! Please try your possesion again later!"
 		return
 	notified = 0
-	brainmob.mind = candidate.mind
```

Comment:
Ah yes, there probably is something in Login() (called at login and when entering a mob) that makes a mind, fair enough.